src/main/java/seedu/task/model/task/Timing.java

```diff
@@ -19,7 +19,8 @@
             "Task timing should be in the format HH:mm dd/MM/yyyy OR dd/MM/yyyy " +
                     "Use only HH:mm if today is the default date";
     public static final String[] TIMING_FORMAT = {"HH:mm dd/MM/yyyy", "dd/MM/yyyy"};
-    public static final String NULL_TIMING = "floating";
```

Comment:
oh good catch. i don't mind either way, just thought the new name was more descriptive.